src/ports/catalog/queries.ts

```diff
@@ -338,6 +358,10 @@ export const getOnlyMintingWhere = () => {
   return SQL`items.search_is_store_minter = true AND available > 0`
 }
 
+export const getOnlyMintingWhereWithTrades = () => {
```

Comment:
This seems to be the same query that we're defining on line 281, what do you think if we re-use it?